client/landing/stepper/declarative-flow/internals/steps-repository/importer-migrate-message/index.tsx

```diff
@@ -37,28 +37,31 @@ const ImporterMigrateMessage: Step = () => {
 	const siteSlugParam = useSiteSlugParam();
 	const fromUrl = useQuery().get( 'from' ) || '';
 	const siteSlug = siteSlugParam ?? '';
-	const isCredentialsSkipped = useQuery().get( 'credentials' ) === 'skipped';
+	const shouldPreventTicketCreation = useQuery().get( 'preventTicketCreation' ) === 'true';
 	const { isPending, sendTicket } = useSubmitMigrationTicket();
 
 	useEffect( () => {
 		recordTracksEvent( 'wpcom_support_free_migration_request_click', {
 			path: window.location.pathname,
 			automated_migration: config.isEnabled( 'automated-migration/collect-credentials' ),
-			skipped_credentials: isCredentialsSkipped,
+			prevent_ticket_creation: shouldPreventTicketCreation,
```

Comment:
should we keep the same value as before (`skipped_credentials`) to not mess with existing funnels? Not sure if those exist, but maybe it's a good idea not to change this

Reply:
I think we should deal with the change in the funnels. Currently, there's no way to know if we skipped the Credentials step in the migrate message step. I think we need to add a new tracks events on the skip event in the Credentials step.

Reply:
Alternatively, we can continue using the skipping flag. But it doesn't sound good to me, as the only purpose of this flag would be for the track events. It seems correct to handle that on the Credentials step.

Reply:
Actually, we can use the `calypso_signup_actions_submit_step` tracks event, by filtering the `action=skip && step=site-migration-credentials`.

Reply:
@valterlorran Ok, so, if I understand correctly, there would be no issue in keeping the same value as before, right?

Reply:
If the funnels you mentioned use this prop that I'm removing, we can update it to use the `calypso_signup_actions_submit_step.`